### DISCUSSION TITLE
Documentation/readme change: read-json-from-a-file section

### DISCUSSION THREAD

**Discussion by Basilisvirus** [nlohmann/json#3736] [Ideas]
in the Readme file at section "read-json-from-a-file ": https://github.com/nlohmann/json#read-json-from-a-file 

The example should include a note that, the created file should not be empty, and must be a .json file with a valid .json context.

```
#include <fstream>
#include <nlohmann/json.hpp>
using json = nlohmann::json;

// ...

std::ifstream f("example.json");//the example.json file should already contain a valid .json text.
json data = json::parse(f); 

```

I tried to compile the above example and I couldn't figure out why it would give me an error. Someone had this issue and the solution is already here: https://github.com/nlohmann/json/discussions/3708

But it would be nice to add it in the example.

**Comment by falbrechtskirchinger** [comment]
There's so much more wrong with the `README` and how it's structured. It's really a bit of a sore spot for me. As a casual user of this library, when I needed to parse JSON once every couple of years, the `README` managed to enrage me every time without fail. ;-)
